sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCDSQueryBenchmark.scala

```diff
@@ -29,9 +30,9 @@ import org.apache.spark.util.Benchmark
 /**
  * Benchmark to measure TPCDS query performance.
  * To run this:
- *  spark-submit --class <this class> <spark sql test jar> <TPCDS data location>
+ *  spark-submit --class <this class> <spark sql test jar> --data-location <TPCDS data location>
```

Comment:
This looks incorrect?

Reply:
sorry, but I missed your point. what's correct?

Reply:
`--data-location <TPCDS data location> [--query-filter Queries to filter]`?

Reply:
aha, thanks. better to add optional parameters here? I like a simple example here.

Reply:
OK. I see.